projects/ngrx-auto-entity/src/lib/factories/load-factories.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { createReducer, on } from '@ngrx/store';
+import { hot } from 'jasmine-marbles';
+import { map } from 'rxjs/operators';
+import { Entity, Key } from '../..';
+import { createLoadAction } from './load-factories';
+
+@Entity('Test')
+class Test {
+  @Key id: number;
+}
+
+describe('createLoadAction()', () => {
```

Comment:
These unit tests prove the compatibility of the new auto-entity action factories with NgRx 8+ `on` and `ofType` functions. Both from a strong typing standpoint, given that TS does not complain about the typing of the reducer function and effects class created in these tests; as well as from an operational standpoint, given the tests pass when run.